### PR TITLE
test(load): add K6 load test workflow with example tests

### PR DESCRIPTION
**Pull request by sethidden**
This commit adds a GitHub Actions workflow (you have to *manually*
trigger it) that uses the K6 load testing utillity to run tests on a
given environment.

I had to add the `K6_API_TOKEN` secret to GitHub to allow running the
tests in K6's cloud (VSF has a paid plan). You can also run the test on
GitHub's agent though.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ module.exports = {
     '@vue-storefront/eslint-config-vue',
     '@vue-storefront/eslint-config-jest',
   ],
+  globals: {
+    "__ENV": "readonly",
```
Comment (sethidden) [inline .eslintrc.js:34]: avoid `no-undef` eslint error

**Comment by sethidden** [comment]
@Frodigo given the tests are pretty ugly because of graphql maybe I could push a PR with just a sample test that queries the environment base url? Like a k6 "hello world"? That way the workflow part would be merged at least.

File: .github/workflows/run-k6-load-test.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      filename:
+        description: The K6 test file to run relative to repository root
+        required: true
+        default: packages/theme/tests/load/searchProduct.js
+        type: string
+
+      environment:
+        description: The full URL of the environment on which load tests will be ran
+        required: true
+        default: https://demo-magento2-dev.europe-west1.gcp.storefrontcloud.io
```
Comment (Frodigo) [inline .github/workflows/run-k6-load-test.yml:20]: It's possible to have something like select here? :) then I will be able to select the environment. If now, please change the default one to canary.

**Review by Frodigo — APPROVED** [review]
Looks good, one question, something good to have.

File: .github/workflows/run-k6-load-test.yml (hunk 2 above)
Comment (sethidden) [reply .github/workflows/run-k6-load-test.yml]: yep, added and changed default value to canary

**Review by Frodigo — APPROVED** [review]

**Pull request merged**